examples/07_nvs_read_write/README.md

```diff
@@ -0,0 +1,7 @@
+# Non-Volatile Storage (NVS) Read and Write Example
+
+Demonstrates how to read and write a value using NVS. The value tracks number of ESP32 module restarts.
+
```

Comment:
In addition to using a fixed size int value, it may be worth demonstrating how to read and write variable-length data (str and blob) using NVS. In particular, demonstrate how to get the length of value, allocate the buffer of the sufficient size, read from nvs into a buffer, and release the buffer. I can imagine that using nvs API for variable length data storage may be less obvious, therefore example would be handy. There is some example code in the header file, it may be worth moving it here.